### PR TITLE
Repo check to see if the wardrobe stuff is out

### PR DESCRIPTION
**Pull request by NopoTheGamer**
upcoming feature 👀

### CODE REVIEW

**Review by jani270 — APPROVED** [review]
erutaef gnimocpu

**Comment by kr45732** [comment]
NotEnoughNopos

File: constants/misc.json [hunk 1]
```diff
@@ -1201,5 +1201,8 @@
       "§6If you read this please ping moulberry on discord and tell him to reopen his Patreon."
     ]
   },
-  "featureslist": "https://gist.github.com/jani270/d33e249d40b0333b87ba5c5e70fca398"
+  "featureslist": "https://gist.github.com/jani270/d33e249d40b0333b87ba5c5e70fca398",
+  "alphawardrobe" : {
```
Comment (IRONM00N) [inline constants/misc.json:1205]: just call it `awardrobe` or something, alpha is redundant

**Review by IRONM00N — CHANGES_REQUESTED** [review]

**Review by jani270 — APPROVED** [review]

File: constants/disable.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
   "auctions": true,
-  "auctions_new": false
+  "auctions_new": false,
+  "wardrobeFeature" : {
```
Comment (IRONM00N) [inline constants/disable.json:4]:
```suggestion
  "disableNewWardrobe": true
```

**Review by IRONM00N — CHANGES_REQUESTED** [review]

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**